pkg/scannerv4/client/client.go

```diff
@@ -100,7 +101,12 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 		connOpt.TLS.UseClientCert = clientconn.MustUseClientCert
 		connOpt.TLS.ServerName = o.serverName
 	}
-	return clientconn.GRPCConnection(ctx, o.mTLSSubject, o.address, connOpt)
+
+	callOpts := []grpc.CallOption{
+		grpc.MaxCallRecvMsgSize(env.ScannerV4MaxRespMsgSize.IntegerSetting()),
+	}
+
+	return clientconn.GRPCConnection(ctx, o.mTLSSubject, o.address, connOpt, grpc.WithDefaultCallOptions(callOpts...))
```

Comment:
The max size the 'server' can receive looks to be `12 MiB`, should the server `MaxRecvMsgSize` (via setting `ROX_GRPC_MAX_MESSAGE_SIZE`) be increased as well to account for the index reports sent to matcher?

Reply:
good point. 64 was a pretty random choice. The index report I ran into that was too large was barely over 4MiB. Instead of increasing the server-side, I'm going to decrease the client side and make sure we document the env vars